week-1/Homework/mandatory/4-tax.js

```diff
@@ -6,10 +6,11 @@
 */
 
 function calculateSalesTax(priceofProduct) {
-  var result=(20/priceofProduct*100);
-  return(result
-  );
-  
+ 
+
+const salesTax=priceofProduct*20/100;
```

Comment:
Again think about using spaces to tidy up your code :)
```suggestion
const salesTax = priceofProduct * 20 / 100;
```